documentatie/use-cases/Invoer-eerste-zitting.md

```diff
@@ -0,0 +1,199 @@
+
+__Niveau:__ hoog-over, vlieger, 🪁
+
+__Precondities:__
+
+- [De beheerder en de coördinator richten de applicatie in](./Installatie-en-inrichting-applicatie.md#de-beheerder-en-de-coördinator-richten-de-applicatie-in-vlieger)
+
+### Hoofdscenario en uitbreidingen
+
+__Trigger:__ Het geplande tijdstip om invoer te starten breekt aan.
+
+__Hoofdscenario:__  
+
+1. De coördinator stelt invoer open.
+2. (voor elk SB PV evt. met corrigendum) [De invoerders vullen de resultaten van de tellingen in.](#de-invoerders-vullen-de-resultaten-van-de-tellingen-in-vlieger)
+3. De coördinator sluit de invoer af.
+4. De applicatie stelt vast dat voor alle stembureaus resultaten zijn ingevoerd.
+5. De applicatie stelt vast dat er geen stembureaus met waarschuwingen zijn.
+6. De applicatie genereert de PVs en EMLs etc.
+
+__Uitbreidingen:__  
+4a. De applicatie stelt vast dat niet voor niet alle stembureaus resultaten zijn ingevoerd:
+
+5a. De applicatie stelt vast dat er stembureaus met geaccepteerde waarschuwingen zijn:
+
+### Open punten
+
+- Moet de applicatie een preview van het gegenereerde PV tonen, zodat de coördinator die kan controleren?
+- Welke controles willen we nog nadat de invoer is afgesloten? Of zijn die controles onderdeel van het afsluiten?
+
+
+## De invoerders vullen de resultaten van de tellingen in (vlieger)
+
+__Niveau:__ hoog-over, vlieger, 🪁
+
+### Hoofdscenario en uitbreidingen
+
+__Trigger:__ De coördinator ontvangt een SB PV evt. met corrigendum.
+
+__Hoofdscenario:__
+
+1. De coördinator geeft het SB PV (eventueel met corrigendum) aan de eerste invoerder.
+2. (tijdens invoer) De coördinator monitort de voortgang op het statusoverzicht van de steminvoer.
+3. [De eerste invoerder voert de resultaten van de telling in.](#de-eerste-of-tweede-invoerder-voert-de-resultaten-van-de-telling-in-zee)
+4. De coördinator geeft het SB PV evt. met corrigendum aan de tweede invoerder.
+5. [De tweede invoerder voert de resultaten van de telling in.](#de-eerste-of-tweede-invoerder-voert-de-resultaten-van-de-telling-in-zee)
+6. (na eerste en/of tweede invoer) De applicatie stelt vast dat de invoer geen waarschuwingen bevat.
+7. De applicatie stelt vast dat beide invoeren gelijk zijn.
+8. De applicatie slaat het definitieve resultaat van het stembureau op.
+
+__Uitbreidingen:__  
+6a. De applicatie stelt vast dat een invoerder waarschuwingen heeft geaccepteerd:  
+&emsp; 6a1. De coördinator beoordeelt de geaccepteerde waarschuwingen.
+
+7a. De applicatie stelt vast dat beide invoeren niet gelijk zijn:  
+&emsp; 7a1. [De coördinator lost de verschillen tussen de twee invoeren op.](#de-coördinator-lost-de-verschillen-tussen-de-twee-invoeren-op-zee)
+
+### Open punten
+- Waar in het scenario kan de coördinator verklaringen maken over de waarschuwingen, zodat het CSB deze beter kan beoordelen? Na de eerste of tweede invoer? Of pas nadat het resultaat van een stembureau definitief is?
```

Comment:
Ik denk dat als je deze optie te laat in het proces presenteert, dus nadat het resultaat definitief is, de coördinator wellicht niet meer helder voor ogen heeft wat het probleem was. Het zou onhandig zijn als de coördinator dat dan in de tussentijd ergens anders moet noteren oid. Daarom zou ik het z.s.m. na de invoer inbouwen.